package.json

```diff
@@ -48,34 +48,45 @@
   "dependencies": {
     "@balena/jellyfish-assert": "^1.2.11",
     "@balena/jellyfish-logger": "4.0.27",
-    "@balena/jellyfish-plugin-base": "^2.2.25",
+    "@balena/jellyfish-mail": "^2.0.169",
+    "@balena/jellyfish-worker": "^16.1.3",
+    "bcrypt": "^5.0.1",
+    "blueimp-md5": "^2.19.0",
+    "googleapis": "^92.0.0",
+    "is-base64": "^1.1.0",
     "is-uuid": "^1.0.2",
     "lodash": "^4.17.21"
   },
   "devDependencies": {
-    "@balena/jellyfish-action-library": "^16.2.0",
     "@balena/jellyfish-config": "^2.0.2",
     "@balena/jellyfish-core": "^14.1.0",
-    "@balena/jellyfish-plugin-product-os": "^2.9.4",
-    "@balena/jellyfish-sync": "^6.3.0",
-    "@balena/jellyfish-test-harness": "^8.1.203",
+    "@balena/jellyfish-plugin-product-os": "^3.0.0",
```

Comment:
Is this dependency still necessary?

Reply:
Sadly yes, because of loops. Maybe there's a way around it, but I get "loop not found" like errors when I don't set up the test context with this plugin.

Reply:
If we actually do need this plugin, we should add it to `requires`. Here's the error I get when I try to set up the test context without the product os plugin:
```
JellyfishNoElement: No such loop: loop-balena-io@1.0.0
```